pytorch_lightning/core/memory.py

```diff
@@ -132,10 +132,7 @@ class ModelSummary(object):
 
     Args:
         model: The model to summarize (also referred to as the root module)
-        mode: Can be one of
-
-             - `top` (default): only the top-level modules will be recorded (the children of the root module)
-             - `full`: summarizes all layers and their submodules in the root module
+        max_depth: Maximum depth of modules to show, or -1 to show all modules.
```

Comment:
So a 0 based value, right? Can you mention it?

Reply:
I'll update it on the next commit along with other suggested changes:
```python
"""
    (...)
    max_depth: Maximum depth of modules to show, starting on 0 (for modules connected root),
                or -1 to show all modules.
    (...)
"""
```